libraries/shared/factories/storage/transformer.go

```diff
@@ -25,11 +25,20 @@ import (
 )
 
 type Transformer struct {
+	Address           common.Address
```

Comment:
Double checking - is it accurate that we'll want a corresponding PR to update all of the initializers for the storage transformers to include this field?

Reply:
Yep, that's correct. Thanks for the reminder!